frontend/src/components/ItemTableRow.tsx

```diff
@@ -0,0 +1,112 @@
+import React, {ChangeEvent, useState} from 'react';
+import {ItemModel} from "../model/ItemModel";
+import axios from "axios";
+import "../css/ItemTableRow.css"
+
+type ItemTableRowProps = {
+    item: ItemModel
+    fetchData: () => void
+    setErrorModal: (showErrorAlert: boolean) => void
+    setErrorMessage: (errorMessage: string) => void
+    setSuccessMessage: (successMessage: string) => void
+    setSuccessModal: (showSuccessAlert: boolean) => void
+}
+
+function ItemTableRow({
+                          item,
+                          fetchData,
+                          setSuccessModal,
+                          setSuccessMessage,
+                          setErrorMessage,
+                          setErrorModal
+                      }: ItemTableRowProps) {
```

Comment:
Destructen von Props ist durchaus üblich. Bislang wurde das in unseren Bootcamps auch so gelehrt.
In unserem Kurs haben wir jetzt eher nicht destructed - ich finde das persönlich etwas übersichtlicher.
Geschmackssache: entscheide selbst, wie Du es besser findest.
```suggestion
function ItemTableRow(props: ItemTableRowProps) {
```